packages/app/src/app/pages/Sandbox/Editor/Workspace/Project/elements.ts

```diff
@@ -4,14 +4,33 @@ import EditPenIcon from 'react-icons/lib/md/create';
 
 import { Link } from 'react-router-dom';
 
-export const Item = styled.div<{ flex?: boolean }>`
+export const Item = styled.div<{ flex?: boolean; css?: any }>`
```

Comment:
@SaraVieira The `css` type prop can be automatically included by adding the following to `packages/app/src/app/pages/Sandbox/Editor/Workspace/index.tsx` (like I did in #1969):

```js
//  Fix css prop types in styled-components (see https://github.com/DefinitelyTyped/DefinitelyTyped/issues/31245#issuecomment-463640878)
import {} from 'styled-components/cssprop'; // eslint-disable-line
```

See https://github.com/DefinitelyTyped/DefinitelyTyped/issues/31245#issuecomment-463640878 for more context